### PR TITLE
Add sysctl to glossary

### PR DESCRIPTION
**Pull request by sftim**
Associated with #5993

### CODE REVIEW

File: content/en/docs/reference/glossary/sysctl.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: sysctl
+id: sysctl
+date: 2019-02-12
+full_link: /docs/tasks/administer-cluster/sysctl-cluster/
+short_description: >
+  An interface for getting and setting kernel parameters
```
Comment (zparnold) [inline content/en/docs/reference/glossary/sysctl.md]:
```suggestion
  An interface for getting and setting linux kernel parameters
```

File: content/en/docs/reference/glossary/sysctl.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+aka:
+tags:
+- tool
+---
+ `sysctl` is a semi-standardized interface for reading or changing the
+ attributes of the running kernel.
```
Comment (zparnold) [inline content/en/docs/reference/glossary/sysctl.md]:
```suggestion
 attributes of the running linux kernel.
```

**Comment by zparnold** [comment]
@sftim Made a small suggestion since we have clusters that run non-Unix style kernels.

**Comment by sftim** [comment]
With tweaks made, is this OK to approve?

**Comment by zparnold** [comment]
Yep, thanks @sftim 
/lgtm
/approve

**Pull request merged**